vector/src/main/java/im/vector/app/features/invite/InviteUsersToRoomActivity.kt

```diff
@@ -101,26 +98,16 @@ class InviteUsersToRoomActivity : SimpleFragmentActivity() {
         viewModel.observeViewEvents { renderInviteEvents(it) }
     }
 
-    private fun openPhoneBook() {
-        // Check permission first
-        if (checkPermissions(PERMISSIONS_FOR_MEMBERS_SEARCH,
-                        this,
-                        PERMISSION_REQUEST_CODE_READ_CONTACTS,
-                        0)) {
-            addFragmentToBackstack(R.id.container, ContactsBookFragment::class.java)
-        }
-    }
-
     override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
         if (allGranted(grantResults)) {
             if (requestCode == PERMISSION_REQUEST_CODE_READ_CONTACTS) {
-                doOnPostResume { addFragmentToBackstack(R.id.container, ContactsBookFragment::class.java) }
+                // TODO. Trigger VM
```

Comment:
TODO?

Reply:
Ups, fixed. Permission logic moved to UserListFragment.